deps/full-os.conf

```diff
@@ -1,8 +1,8 @@
 ARTIFACT_BASE_URL=https://deps.runfinch.com
 
 # From https://dl.fedoraproject.org/pub/fedora/linux/releases/40/Cloud/aarch64/images/
-AARCH64_ARTIFACT=Fedora-Cloud-Base-40-1.14.aarch64-20250117000006.qcow2
-AARCH64_512_DIGEST=915de707f5b1e1d2dab0133f863cd51c70674fca922573f0d9e632b0ea612d36a429e1b2c20037bdc1d024a43520152561139ab34942314b291e004ec94a0392
+AARCH64_ARTIFACT=15:42
+AARCH64_512_DIGEST=
```

Comment:
Did update-os-image.sh do this?

Reply:
Yes. I don't know why. But I have some ideas to improve the process.

Reply:
Yeah, would be great to understand what happened there. The mechanism isn't perfect, but has been pretty reliable so far.